elfi/methods/bo/acquisition.py

```diff
@@ -246,6 +246,161 @@ def evaluate_gradient(self, x, t=None):
         return grad_mean - 0.5 * grad_var * np.sqrt(self._beta(t) / var)
 
 
+class MaxVar(AcquisitionBase):
+    r"""The maximum variance acquisition method.
+
+    The next evaluation point is acquired in the maximiser of the variance of
+    the approximate posterior as defined in [2], i.e.:
+    \theta_{t+1} = arg max Var(p_t(\theta | X_t)).
```

Comment:
I'm sorry, but this is still vague, and it's important to state clearly the assumptions of this method. How about:

The next evaluation point is acquired in the maximiser of the variance of the approximate posterior

\theta_{t+1} = arg max Var(p(\theta) * p_t(X_t | \theta))

where the likelihood is defined as:

p_t(X_t | \theta) \propto \Phi ((\epsilon - \mu_t(\theta)) / \sqrt(\sigma_t^2(\theta) + \sigma_n^2)

where \epsilon is the discrepancy threshold, \mu_t and \sigma_t are determined by the Gaussian process, and \sigma_n determines the noise. [Not sure if we should include \sigma_n separately, since it's implicit inGPy.]

(typos likely)

Reply:
Thanks for the write-up, now it's much clearer for me how you wanted to frame it. About your last note on the noise term, \sigma_n, it would be a good idea to include it in the documentation as we might have alternative `target_model`s in the future w/o implicit noise treatment.